data/css/modules/card/_base.scss

```diff
@@ -38,14 +38,23 @@
       min-width: 96px;
       min-height: 96px;
       border-radius: 50%;
+      color: rgba(white, .6);
       background-color: rgba(#1e1e1e, .8);
```

Comment:
There is no spec for this, but Kate suggested using this very similar one: https://endlessm.invisionapp.com/share/9GC3BQEHQ#/screens/190153077_Video_Player_States_r3-1
That would mean that the background color should be adjusted slightly.